webapp/src/actions/index.js

```diff
@@ -11,6 +11,7 @@ export function startMeeting(channelId) {
             await Client.startMeeting(channelId, true);
         } catch (error) {
             let m = 'We could not verify your Mattermost account in Zoom. Please ensure that your Mattermost email address matches your Zoom email address.';
+            m += '\n If OAuth is enabled, please use the following link to authenticate:';
```

Comment:
@hanzei Can we make this a conditional error?  

If they have OAuth enabled display "To start Zoom meetings from Mattermost, Please use the following link to connect your Zoom account."

If no match based on username (no Oauth): 'We could not verify your Mattermost account in Zoom. Please ensure that your Mattermost email address matches your Zoom email address.'

Reply:
I have handled this conditional message at server side. client parses and displays the error message sent from server